### DISCUSSION TITLE
页面网格布局新增五列模式。

### DISCUSSION THREAD

**Discussion by MJSummer** [BewlyBewly/BewlyBewly#1125] [Ideas]
如题。即每行展示五个视频，能够更加适合27寸2k显示器的浏览体验。

**Comment by hakadao** [comment]
現在不是你寬屏的話就顯示5個嗎，自適應的

**Reply by MJSummer** [reply]
横向素2560，但是还是4个。
![image](https://github.com/user-attachments/assets/5eb268ea-33cc-415b-aba6-cc0972c7d8ff)

**Reply by hakadao** [reply]
https://support.microsoft.com/zh-cn/windows/放大文本和应用-c3095a80-6edd-4779-9282-623c4d721d64

請查閱「在屏幕上放大图像和应用」，適當把縮放調小
